cmd/kubeadm/app/cmd/reset.go

```diff
@@ -152,7 +152,7 @@ func drainAndRemoveNode(removeNode bool) error {
 	hostname = strings.ToLower(hostname)
 
 	// TODO: Use the "native" k8s client for this once we're confident the versioned is working
-	kubeConfigPath := filepath.Join(kubeadmapi.GlobalEnvParams.KubernetesDir, kubeconfig.KubeletKubeConfigFileName)
```

Comment:
Sorry, but this won't work.
`kubeadm reset` should work on every node, and admin.conf only exists on the master.